fair/tools/unifiedtools.py

```diff
@@ -196,6 +199,86 @@ def create_output_dataframes(inp_df, gas_np, RF_np, T_np, alpha_np, ext_forcing_
     )
     return gas_df, RF_df, T_df, alpha_df
 
+def create_output_dataframe_aimc_compliant(inp_df, gas_np, RF_np, T_np, alpha_np, ext_forcing_np):
+    """
+    TODO: docstring
+    """
+
+    units = Units()
+
+    variable_array = inp_df.timeseries().index.levels[3].to_numpy()
+    split_variable_array = np.array([variable_array[i].split('|',1) for i in range(len(variable_array))])
+
+    gas_array = split_variable_array[:,1]
+    function_array = split_variable_array[:,0]
+    unique_function_array = np.unique(function_array)
+    if len(unique_function_array) > 1:
+        raise Exception('Error: More than one type of input passed')
+
+    inp_function = unique_function_array[0]
+
+    output_function = "Atmospheric Concentrations" if str.lower(inp_function[0]) == 'e' else "Emissions"
+
+    model_region_scenario_array = np.unique(inp_df.timeseries().index.droplevel(('unit','variable')).to_numpy())
+    if len(model_region_scenario_array) > 1:
+        raise Exception('Error: More than one Model, Region + Scenario combination input passed')
```

Comment:
```suggestion
        raise ValueError('More than one Model, Region + Scenario combination input passed')
```

??

Something more specific than the very general Exception would be great